cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/azure-orchestrator/AzureOrchestrator.service.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { AzureService, AzureVirtualMachine } from '../../../../../swagger-api';
+import { AZURE_FIELDS } from '../../azure-mc-basic/AzureManagementClusterBasic.constants';
+import { INPUT_CHANGE } from '../../../../../state-management/actions/Form.actions';
+import { AzureDefaults } from '../default-service/AzureDefaults.service';
+import { AzureResourceAction, FormAction, StoreDispatch } from '../../../../../shared/types/types';
+import { AZURE_ADD_RESOURCES } from '../../../../../state-management/actions/Resources.actions';
+
+interface AzureOrchestratorProps {
+    azureState: { [key: string]: any };
+    azureDispatch: StoreDispatch;
+    errorObject: { [key: string]: any };
+    setErrorObject: (newErrorObject: { [key: string]: any }) => void;
+}
+export class AzureOrchestrator extends React.Component {
```

Comment:
Are you able to create this class without extending React.Component? I don't think the orchestrators and defaults services need this since we aren't using lifecycle hooks or render. I might be missing something, so please let me know if I am wrong.

Reply:
Oh oh. Yes sir. You are right. This comment is the same with previous _AwsDefault_ PR.
We do not need extend React.Component actually. 
I created this branch in the same time when I created the AwsOrchestrator. Sorry I forgot to remove the react import in this branch.